examples/ct/structures.go

```diff
@@ -121,6 +121,38 @@ func SignV1SCTForPrecertificate(km crypto.KeyManager, cert *x509.Certificate, t
 	return serializeAndSignSCT(km, leaf, sctInput, t)
 }
 
+// SignV1TreeHead signs a tree head for CT. The input STH should have been built from a
+// backend response and already checked for validity.
+func SignV1TreeHead(km crypto.KeyManager, sth *ct.SignedTreeHead) error {
```

Comment:
Just thinking that since this is a toy example it might be actively dangerous for someone to reuse bits of this to build a 'real' log?